app/src/main/java/org/mifos/mobile/models/entity/Transaction.kt

```diff
@@ -0,0 +1,14 @@
+package org.mifos.mobile.models.entity
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Transaction(
+        @SerializedName("payee")
```

Comment:
we can remove `SerializedName` annotation, by default field name should be picked. Please confirm. `[applies across all models]`

Reply:
Should I remove the annotations only from those fields which have the same `field name` and `SerializedName` or should I remove the notations completely from all the models and make the field name exactly same to the `SerializedName`. Please tell.

Reply:
Depends on the scenario.

Usually, how I would deal with this scenario, if the API which is gonna consume fields follow `camelcase convention`, I simply use field name. (annotation not needed) 
Sometimes, API's field name doesn't follow `camelcase` (for instance, like ABC_PAYEE or abc_payee or any other regex). Then I prefer `SerializedName` since we prefer `camelcase` as standard in our repos.

Hope it answers your question.

Reply:
I got your point completely. I'll update the models accordingly.